src/utils/userLocation.ts

```diff
@@ -1,52 +1,83 @@
-import firebase, { UserInfo } from "firebase/app";
+import firebase from "firebase/app";
 
 import { AnyRoom } from "types/rooms";
 
 import { updateUserProfile } from "pages/Account/helpers";
 import { useInterval } from "hooks/useInterval";
 
 import { getCurrentTimeInMilliseconds } from "./time";
-import { openRoomUrl, openUrl, venueInsideUrl } from "./url";
+import { openRoomUrl } from "./url";
 
 const LOCATION_INCREMENT_SECONDS = 10;
 const LOCATION_INCREMENT_MS = LOCATION_INCREMENT_SECONDS * 1000;
 
 type LocationData = Record<string, number>;
 
-export const updateLocationData = (
-  userId: string,
-  newLocationData: LocationData = {}
-) => {
+type UpdateLocationDataProps = {
```

Comment:
```suggestion
export type UpdateLocationDataProps = {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1129/commits/53f57c40e24983470b7552d4ebed6f1715ff74ba